torchci/rockset/commons/__sql/recent_pr_workflows_query.sql

```diff
@@ -36,12 +37,13 @@ FROM
   JOIN commons.workflow_run w ON w.id = j.run_id
 UNION
 SELECT
-  null AS workflow_name,
+  null AS workflow_id,
```

Comment:
Discover this bug, this should be called `workflow_id` instead to match with the the other side of the UNION